Uchu.World/Objects/ItemSet.cs

```diff
@@ -91,6 +91,27 @@ private void SetupEvents()
         /// The items in the inventory that are equipped and part of this item set
         /// </summary>
         private readonly HashSet<Lot> _equippedItemsInSet;
+        public int EquippedItemCount 
```

Comment:
You can skip 6 unnecessary lines by writing this as `public int EquippedItemCount => _equippedItemsInSet.Count;`, same goes for `SetID` and `ItemsInSet`